Sources/CloudyKit/Predicate+Helpers.swift

```diff
@@ -58,16 +68,16 @@ public class CKPredicate {
 
 extension Predicate {
     
-    var filterBy: [CKWSFilterDictionary]? {
+    func filterBy() throws -> [CKWSFilterDictionary]? {
```

Comment:
I'd like to keep the `fatalError` calls because Apple throws an exception when using an invalid predicate.